tests/pytype_blacklist.txt

```diff
@@ -1,13 +1,116 @@
-# Pytype blacklist. Files will not be tested with pytype.
+# Pytype blacklist. Files will not be tested with pytype. Notice that the parse
+# test will work for some of the files. Loading the files will fail, but is not
```

Comment:
Is it possible to make typeshed's pytype tests also load these files? It's not ideal if the typeshed project has to maintain a blacklist file without a test to ensure the blacklist is correct.

Reply:
I agree. This will take a few more days. The code path to easily test this is not yet in PyType's github repo.